Form/Type/HoneypotType.php

```diff
@@ -58,7 +58,7 @@ public function __construct(RequestStack $requestStack, HoneypotManager $honeypo
     /**
      * {@inheritdoc}
      */
-    public function buildForm(FormBuilderInterface $builder, array $options)
+    public function buildForm(FormBuilderInterface $builder, array $options): void
```

Comment:
@eymengunay adding return types in non-final classes is actually a bc break if it's done in a minor. This currently break applications extending this class after the update to 2.2.0